code/modules/recycling/disposal.dm

```diff
@@ -514,10 +514,10 @@
 			if(prob(75))
 				I.forceMove(src)
 				for(var/mob/M in viewers(src))
-					M.visible_message("\The [I] lands in \the [src].", 3)
+					M.visible_message("\The [I] lands in \the [src].")
 			else
 				for(var/mob/M in viewers(src))
-					M.visible_message("\The [I] bounces off of \the [src]\'s rim!", 3)
+					M.visible_message("\The [I] bounces off of \the [src]\'s rim!")
```

Comment:
You don't need a for viewer in visible message

Reply:
Haven't even touched that.

![image](https://user-images.githubusercontent.com/65828539/164638341-b48270a9-bb67-4bf7-8f17-8886b6c06003.png)

Reply:
It works with show message, as it shows message to each mob specifically, but visible message already does that, so you don't need to loop over every mob (it would (and does) make a message for each mob there is for every other mob)

Reply:
I blame Kegdo.